srecord/input/generator.cc

```diff
@@ -107,7 +107,7 @@ srecord::input_generator::create(srecord::arglex_tool *cmdln) -> srecord::input:
                 cmdln->get_number("--generate --b-e-constant <length>");
             if (length <= 0 || length > 4)
             {
-                cmdln->fatal_error
+                srecord::arglex_tool::fatal_error
```

Comment:
These appear to be a consequence of earlier change to make things static that previously weren't....

However, the check is valid and the resulting change necessary.